astrocut/tests/test_cube_cut.py

```diff
@@ -231,71 +243,119 @@ def test_cutout_extras(tmp_path, ffi_type):
         assert ext1_header['BJDREFI'] == ext2_header['BJDREFI'] == primary_header['BJDREFI']
 
 
-    ############################
-    # Test  _get_cutout_limits #
-    ############################
+@pytest.mark.parametrize("ffi_type", ["SPOC", "TICA"])
+def test_get_cutout_limits(cube_file, ffi_type, tmp_path):
+    """Test _get_cutout_limits"""
+
+    tmpdir = str(tmp_path)
+
+    cutout_maker = CutoutFactory()
+
+    # Making the cutout
+    coord = "256.88 6.38"
+    cutout_size = [5, 3]
+    out_file = cutout_maker.cube_cut(
+        cube_file, coord, cutout_size, ffi_type, output_path=path.join(tmpdir, "out_dir"), verbose=False
+    )
+
     xmin, xmax = cutout_maker.cutout_lims[0]
     ymin, ymax = cutout_maker.cutout_lims[1]
-    
+
     assert (xmax-xmin) == cutout_size[0]
     assert (ymax-ymin) == cutout_size[1]
 
     cutout_size = [5*u.pixel, 7*u.pixel]
     out_file = cutout_maker.cube_cut(cube_file, coord, cutout_size, ffi_type, verbose=False, output_path=tmpdir)
     assert "256.880000_6.380000_5x7_astrocut.fits" in out_file
-    
+
     xmin, xmax = cutout_maker.cutout_lims[0]
     ymin, ymax = cutout_maker.cutout_lims[1]
-    
+
     assert (xmax-xmin) == cutout_size[0].value
     assert (ymax-ymin) == cutout_size[1].value
 
     cutout_size = [3*u.arcmin, 5*u.arcmin]
     out_file = cutout_maker.cube_cut(cube_file, coord, cutout_size, ffi_type, verbose=False, output_path=tmpdir)
     assert "256.880000_6.380000_8x15_astrocut.fits" in out_file
-    
+
     xmin, xmax = cutout_maker.cutout_lims[0]
     ymin, ymax = cutout_maker.cutout_lims[1]
-    
-    assert (xmax-xmin) == 8
-    assert (ymax-ymin) == 15
-    
-    # todo: figure out what this is testing and re-enable
-    # cutout_size = [1*u.arcsec, 5*u.arcsec]
-    # out_file = cutout_maker.cube_cut(cube_file, coord, cutout_size, ffi_type, verbose=False, output_path=tmpdir)
-    # assert "256.880000_6.380000_1x1_astrocut.fits" in out_file
-    
-    # xmin, xmax = cutout_maker.cutout_lims[0]
-    # ymin, ymax = cutout_maker.cutout_lims[1]
-    
-    # assert (xmax-xmin) == 1
-    # assert (ymax-ymin) == 1
 
-    #############################
-    # Test _get_full_cutout_wcs #
-    #############################
-    cutout_size = [5, 3]
+    assert (xmax - xmin) == 8
+    assert (ymax - ymin) == 15
+
+
+# todo: figure out what this is testing and re-enable
```

Comment:
Should remove this `todo` comment

Reply:
yup, that's the plan, once i figure out what the problem is with it!

Reply:
Oh, I thought it was uncommented because it works now

Reply:
no, it still does not work. it's marked with xfail

Reply:
ah yep, see that now. thanks